cql3/Cql.g

```diff
@@ -447,14 +447,15 @@ selector returns [shared_ptr<raw_selector> s]
 unaliasedSelector returns [shared_ptr<selectable::raw> s]
     @init { shared_ptr<selectable::raw> tmp; }
     :  ( c=cident                                  { tmp = make_shared<selectable::with_expression::raw>(cql3::expr::unresolved_identifier{std::move(c)}); }
-       | K_COUNT '(' countArgument ')'             { tmp = selectable::with_function::raw::make_count_rows_function(); }
+       | K_COUNT '(' countArgument ')'             { tmp = ::make_shared<selectable::with_expression::raw>(make_count_rows_function_expression()); }
```

Comment:
(not a review) /cc @havaker - keep track of these changes, it's probably going to change the way we can recognize if a request is an aggregate